docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+* `java.lang.Int`  => `Date`
+* `java.lang.Int` => `Time (millisecond precision)`
+* `java.lang.Long` => `Time (microsecond precision)`
+* `java.lang.Long` => `Timestamp (millisecond precision)`
+* `java.lang.Long` => `Timestamp (microsecond precision)`
+
+If any value will be not the proper logical type (e.g. "str" => UUID) then an exception on runtime will be thrown.
+
+## JSON Schema
+
+To handle JSON Schema Nussknacker uses `everit-json-schema` in version `1.14.1`, and this lib provides some inconsistency against JSON Schema, e.g.
+
+* integer schema should [accept value 1.0](https://json-schema.org/understanding-json-schema/reference/numeric.html#integer) but
+  everit validation throws an exception. This issue occurs only on deserialization data (source), on serialization (sink) we handled it.
+
+[//]: # (See SwaggerBasedJsonSchemaTypeDefinitionExtractor)
+Nussknacker doesn't support:
+* Nested relative schemas
+* Recursive schemas
+* Anchors
+
+### Source supports & conversions
+
+* Primitive types: all with some conversions
+  * [Integer](https://json-schema.org/understanding-json-schema/reference/numeric.html#integer) => `java.lang.Long`
+    * If we pass number bigger then `java.lang.Long.MAX_VALUE`, value will be rounded to `java.lang.Long.MAX_VALUE`
+    * TODO: Support for [range](https://json-schema.org/understanding-json-schema/reference/numeric.html#range) and do proper conversion to: Int/Long/BigInteger
+  * [Number](https://json-schema.org/understanding-json-schema/reference/numeric.html#number) => `java.math.BigDecimal`
+    * TODO: Support for [range](https://json-schema.org/understanding-json-schema/reference/numeric.html#range) and do proper conversion to: Float/Double/BigDecimal
+  * [String](https://json-schema.org/understanding-json-schema/reference/string.html#format) with [formats](https://json-schema.org/understanding-json-schema/reference/string.html#format):
+    * `date-time` => `java.time.ZonedDateTime`
+    * `date` => `java.time.LocalDate`
+    * `time` => `java.time.OffsetTime`
+* [Enum](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) => `java.lang.String`
+* `Object` => `java.util.HashMap`
+  * Static properties (fields) typing
+  * `additionalProperties` are supported, but additional fields won't be available during the hinting on the Designer. 
+  To get add field you have to do `#inpute.get("additional-field")`, but remember the result of this expression will be `Unknown`.
+  * Object without properties and `additionalProperties` is treated as `Map[String, Type]`, where `Type` is depends on
+  `additionalProperties` configuration: `true` means `Unknown`, `{"type": }` will be converted to a specific type.
+* Array
+* [Schema Composition](https://json-schema.org/understanding-json-schema/reference/combining.html)
+  * Nussknacker supports just `anyOf` and `oneOf` to build static typing information about the field. 
+
+### Sink validation & encoding
```

Comment:
Stuff below looks like a collection of helpful links; I do not see the connection to the section title. All links are to external stuff, so the content there will not explain how Nussknacker works.